src/main/java/net/visma/autopay/http/signature/SignatureContext.java

```diff
@@ -182,6 +189,28 @@ public Builder targetUri(String targetUri) {
             return this;
         }
 
+        /**
+         * Sets target URI
+         * <p>
+         * Method offers compatibility with {@code HttpServletRequest} methods.
+         * {@code targetUri} is not the same as servlet's {@code requestUri}.
+         *
+         * @param requestUrl  request URL
+         * @param queryString query string
+         * @return This builder
+         */
+        public Builder targetUri(StringBuffer requestUrl, String queryString) {
```

Comment:
Sonar complains about using `StringBuffer`: https://sonar.visma.com/project/issues?resolved=false&inNewCodePeriod=true&types=CODE_SMELL&pullRequest=66&id=AutoPay.HttpSignatures&open=AYzZ-Y3eSVBfSUagLExq

I think we can suppress it (in the code).